numpy-skips.txt

```diff
@@ -1,11 +0,0 @@
-# These tests cause a core dump on CI, so we have to skip them entirely
-array_api_tests/test_operators_and_elementwise_functions.py::test_floor_divide[__floordiv__(x, s)]
-array_api_tests/test_operators_and_elementwise_functions.py::test_floor_divide[__floordiv__(x1, x2)]
-array_api_tests/test_operators_and_elementwise_functions.py::test_floor_divide[__ifloordiv__(x, s)]
-array_api_tests/test_operators_and_elementwise_functions.py::test_floor_divide[__ifloordiv__(x1, x2)]
-array_api_tests/test_operators_and_elementwise_functions.py::test_floor_divide[floor_divide(x1, x2)]
-array_api_tests/test_operators_and_elementwise_functions.py::test_remainder[__imod__(x, s)]
-array_api_tests/test_operators_and_elementwise_functions.py::test_remainder[__imod__(x1, x2)]
-array_api_tests/test_operators_and_elementwise_functions.py::test_remainder[__mod__(x, s)]
-array_api_tests/test_operators_and_elementwise_functions.py::test_remainder[__mod__(x1, x2)]
-array_api_tests/test_operators_and_elementwise_functions.py::test_remainder[remainder(x1, x2)]
```

Comment:
Tested these and could reproduce none of the core dumps

Reply:
Can't say I fancy chasing segfaults down the line. OTOH we won't be able to remove these skips otherwise, so OK, let's live dangerously and see if these reappear.